src/uct/ib/dc/dc_mlx5.c

```diff
@@ -103,6 +103,11 @@ ucs_config_field_t uct_dc_mlx5_iface_config_sub_table[] = {
      ucs_offsetof(uct_dc_mlx5_iface_config_t, fc_hard_req_timeout),
      UCS_CONFIG_TYPE_TIME_UNITS},
 
+    {"MAX_DCI_STREAM_CHANNELS", "1",
+     "Maximum DCI stream channels per DCI to be used in DCI-rand mode",
```

Comment:
"Number of stream channels per DCI to be used in DCI-rand mode. A value of 1 disables"
"DCI multi-channel support." ,

Reply:
it's the same QP but can use multiple "engines" in the underlying implementation

Reply:
So you don't have to mange QP pull anymore ?

Reply:
Don't have to. But we still want to support older HW generations in the code.

Reply:
Very cool.